packages/contracts/src/dollar/utils/TransferHelper.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
```

Comment:
We already have this safe, and it's imported from openzeppelin for ref check LibDirectGovernanceImplementation where it uses using SafeERC20 for ERC20 and just use the ERC20 type

Reply:
Fixed